sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubProducerOptions.cs

```diff
@@ -7,24 +7,24 @@
 namespace Azure.Messaging.EventHubs
```

Comment:
There is a spelling mistake, "considered"
https://github.com/jsquire/azure-sdk-for-net/blob/2568fe0c58b2117904e857c0faff34c8e440fe04/sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubProducerOptions.cs#L52

Reply:
Thank you!